types/src/account_config/resources/coin_info.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Aptos
```

Comment:
maybe we should add some comments to this file, I was super confused about introducing a new Aggregator struct and the limited functionalities